rfcs/0000-rfc-process/text.md

```diff
@@ -55,18 +52,17 @@ If the change is major, the following are the steps to revising an existing RFC:
 2. Make your changes in the appropriate RFC. If there are new approvers, **append** them to the list of approvers at the top of the RFC.
 3. Create a pull request. This is probably the most important step and most different step from creating a branch new RFC. In the top level comment of the pull request, ensure that you succinctly summarize the changes. The commit message should be a complete summary of the changes and why there were made. Also note in the top level comment of the RFC who the approvers are of the specific pull request. The thoroughness in the commit messages and top level comment is required so that the history of the RFC is much more easily accessible and comprehensible.
 4. As a courtesy, post a heads up on the [#org-sci-eng-single-cell](https://chanzuckerbergteam.slack.com/archives/GQGPP7925) Slack channel that there is a design modification available for review, specifically calling out the approvers. Use your best judgement on how much time you'd like for your modification to be under review, but roughly should be anywhere from a couple days to 2 weeks, depending on the size of the change. Note that huge changes like re-architecting probably warrants its own RFC.
-5. Once all comments are addressed and the *approvers* have approved your pull request, merge the PR!
+5. Once all comments are addressed and the _approvers_ have approved your pull request, merge the PR!
 6. Delete your branch.
 
-
 ## Alternatives
 
 This process was initially done on Google Docs. As mentioned earlier, the process didn't work out well for keeping track of modifications to the RFC as versioning was not easy to use. Other ideas that was considered include [IETF's Github process](https://tools.ietf.org/html/draft-ietf-git-using-github-06) as well as the [Data Coordination Platform RFC process](https://github.com/HumanCellAtlas/dcp-community/blob/master/rfcs/text/0001-rfc-process.md).
 
 ## References
 
-[0] [IETF's Github Process](https://tools.ietf.org/html/draft-ietf-git-using-github-06)
+[0][ietf's github process](https://tools.ietf.org/html/draft-ietf-git-using-github-06)
```

Comment:
Why are these lowercased?

Reply:
And ditto in lines below?

Reply:
Ahh sorry, looks like a bug in `prettier` 🤦 

https://github.com/prettier/prettier/issues/3835

I merged the change in https://github.com/chanzuckerberg/single-cell/pull/67/files already, but I'll fix it in a follow up PR now. Sorry!

Reply:
https://github.com/chanzuckerberg/single-cell/pull/68 PR out! I'll resolve the conflict in this PR once that one merges into `main`! Thank you!

Reply:
@maniarathi PTAL at the PR 😄🙏 Thank you!

https://github.com/chanzuckerberg/single-cell/pull/68

Reply:
Done, thank you!